libs/comments/website/src/lib/comment-editor/comment-editor.spec.tsx

```diff
@@ -1,13 +1,23 @@
 import {render} from '@testing-library/react'
 import * as stories from './comment-editor.stories'
 import {composeStories} from '@storybook/react'
+import {ThemeProvider} from '@mui/material/styles'
+import {MockedProvider} from '@apollo/client/testing'
+import {theme} from '@wepublish/ui'
 
+// Compose stories
 const storiesCmp = composeStories(stories)
 
 describe('CommentEditor', () => {
   Object.entries(storiesCmp).forEach(([story, Component]) => {
     it(`should render ${story}`, () => {
-      const {asFragment} = render(<Component />)
+      const {asFragment} = render(
+        <MockedProvider>
```

Comment:
ThemeProvider is already in component and don't think we need the ApolloProvider here, do we?

Reply:
after removing the tests are failing